bin/pycbc_multi_inspiral

```diff
@@ -41,7 +41,7 @@ from pycbc.filter import MatchedFilterControl
 from pycbc.types import TimeSeries, zeros, float32, complex64
 from pycbc.types import MultiDetOptionAction
 from pycbc.vetoes import sgchisq
-
+from pycbc.conversions import mchirp_from_mass1_mass2
```

Comment:
Please move this up for code style: we were following alphabetical order on the submodule name.

Reply:
I thought we agreed in previous PRs/discussions that we do not want to store template parameters with the triggers?